devel_requirements.txt

```diff
@@ -1,5 +1,8 @@
 black
-nypl-py-utils[avro-client,config-helper]==1.2.0
+certifi
+idna
```

Comment:
what are these?

Reply:
These packagese caused the lambda_function to fail until I imported them. I think they're both dependencies that were originally imported from nypl-py-utils until I added the `--no-dependencies` flag (since I trying to avoid importing pandas). Pandas was being installed regardless of what packages I was installing from nypl-py-utils, even if the packages didn't use pandas (e.g. avro-client)

Reply:
I tested locally and I think pandas is coming from `python-csv` not `nypl-py-utils`. You could not use `python-csv` and use the regular `csv` module instead or you could remove `python-csv` from the requirements, add it to devel_requirements, and install it separately with `--no-dependencies` in the deployment script

Reply:
Ok everything is working now, I removed python-csv and that resolved all the issues LOL